docs(using-atlantis): add usage for -destroy flag in atlantis plan (#3755)
runatlantis/atlantis: runatlantis.io/docs/using-atlantis.md

```
If you always need to append a certain flag, see [Custom Workflow Use Cases](custom-workflows.html#adding-extra-arguments-to-terraform-commands).

### Using the -destroy Flag

#### Example

> To perform a destructive plan that will destroy resources in the specified directory, you can use the `-destroy` flag like this:

Reviewer: ```suggestion
To perform a destructive plan that will destroy resources you can use the `-destroy` flag like this:
```

```bash

> atlantis plan -d dir -- -destroy

Reviewer: ```suggestion
atlantis plan -- -destroy
```

Author: There are cases where users may run on a different directory, so I think `atlantis plan -d dir -- -destroy` this could also be useful for new users to have

```
::: warning NOTE 

> The `-destroy` flag initiates resource destruction, which can result in data loss or service disruptions. Ensure that you have thoroughly reviewed your Terraform configuration and intend to remove the specified resources before using this flag.

Reviewer: ```suggestion
The `-destroy` flag generates a destroy plan, If this plan is applied it can result in data loss or service disruptions. Ensure that you have thoroughly reviewed your Terraform configuration and intend to remove the specified resources before using this flag.
```
